crates/biome_css_analyze/src/assist/source/use_sorted_properties.rs

```diff
@@ -116,9 +116,7 @@ impl Rule for UseSortedProperties {
             .into_iter()
             .collect::<Vec<AnyCssDeclarationOrRule>>();
 
-        if contains_shorthand_after_longhand(&original_properties)
-            || contains_unknown_property(&original_properties)
-        {
+        if contains_shorthand_after_longhand(&original_properties) {
```

Comment:
The original intended behavior for this was that unknown properties shouldn't cause an error on their own, but if a block has other out of order properties as well as unknown properties, then the unknown ones are placed at the end. That's what the the stylelint rule does, but I guess I broke it.

If you want to change it to skip blocks with unknown properties, couldn't you change this case to `return None`? (Instead of sorting first, and then checking for unknown properties.)

Reply:
i will change it back~